### PR TITLE
[Merged by Bors] - fix: name collision on Countable.toSmall

### PR DESCRIPTION
**Pull request by digama0**
This is currently causing mathport to fail, it's a new kind of name collision that has not occurred before: two definitions made in two different files are being `#align`ed to the same thing. Normally mathport will automatically rename one of them but in this case neither one imports the other so it doesn't know to avoid the situation.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Countable/Small.lean [hunk 1]
```diff
@@ -22,7 +22,7 @@ instance (priority := 100) Countable.toSmall (α : Type v) [Countable α] : Smal
   let ⟨_, hf⟩ := exists_injective_nat α
   small_of_injective hf
 #align small_of_countable Countable.toSmall
-#align small_of_fintype Countable.toSmall
+#align small_of_fintype Countable.toSmallₓ
```
Comment (jcommelin) [inline Mathlib/Data/Countable/Small.lean:25]:
```suggestion
#align small_of_fintype Countable.toSmallₓ -- already aligned elsewhere
```

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Mathlib/Data/Countable/Small.lean (hunk 1 above)
Comment (digama0) [reply Mathlib/Data/Countable/Small.lean:25]:
```suggestion
#align small_of_fintype Countable.toSmallₓ -- this alignment clashes with the one above
```

**Pull request closed without merging**